components/EditProfilePage/TestimonyFAQ.tsx

```diff
@@ -8,7 +8,7 @@ const StyledCard = styled(Card)`
   background: var(--bs-blue);
   border: none;
   color: white;
-  font-family: Nunito;
+  /* font-family: Nunito; */
```

Comment:
I thought Nunito is inherited now; can possibly just eliminate this line altogether?